api.txt

```diff
@@ -0,0 +1,53 @@
+Disks() - constructor
+
+get_partitions() - generator yielding following get_partitions
```

Comment:
the `api.txt` needs to be updated
`get_partitions()` is no longer a generator, `File` has other constructor, etc :p